### PR TITLE
add function to get raw slot

### PR DESCRIPTION
**Pull request by parithosh**

### CODE REVIEW

**Comment by barnabasbusa** [comment]
We need to have this script somewhere central too. I'm sick of updating it everywhere else

**Comment by parithosh** [comment]
kinda agreed :D
we can just toss a per testnet env and pull the script from s3 or whatever :D

**Comment by barnabasbusa** [comment]
put it in its own repo, and make releases, and publish it in brew?

**Comment by parithosh** [comment]
git submodule?

**Comment by barnabasbusa** [comment]
Requirement: I wanna be able to edit the script from phone, possibly run the script from phone

File: scripts/run.zsh [hunk 1]
```diff
@@ -97,6 +98,20 @@ for arg in "${command[@]}"; do
...
       ;;
+    "get_slot")
+      if [[ -z "${command[2]}" ]]; then
+        echo "Please provide a slot number as the second argument, or get the latest slot"
+        echo "  Example: ${0} get_slot 100"
+        ${0} latest_slot
+        exit;
+      else
+        slot=${command[2]}
+        # Get the latest slot of the network
```
Comment (barnabasbusa) [inline scripts/run.zsh:109]: this comment is not right. It will get a specific slot that you use as a variable.

**Comment by parithosh** [comment]
lol, just walk to your laptop and run it there :P

File: scripts/run.zsh (hunk 1 above)
Comment (parithosh) [reply scripts/run.zsh]: ehh isn't the correct? i want to get the slot i specify and default is latest :D
updated comment

**Comment by parithosh** [comment]
can you actually ever run it from your system? we rely on ethdo for some calls and that uses docker, can iphone run docker? or install random go binaries?

**Comment by barnabasbusa** [comment]
no 😢

**Review by barnabasbusa — APPROVED** [review]

**Comment by parithosh** [comment]
loool, time to get an android phone finally? super easy guide: https://gist.github.com/FreddieOliveira/efe850df7ff3951cb62d74bd770dce27

**Pull request merged**